<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadingWindow</class>
 <widget class="QWidget" name="LoadingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>203</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar:horizontal {
border: 2px solid #FFFFFF;
border-radius: 3px;
background: #7DA3FD;
padding: 1px;
text-align: center;
}
QProgressBar::chunk:horizontal {
background: #FFFFFF;
}</string>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>275</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#background{
border-radius: 10px;
background: #7DA3FD;
border: 3px solid;
border-color: #DBD9D9;

}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="close_bu">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>10</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid #707070;
border-radius: 10px;
background:#DD6243;
color: #707070;
}
QPushButton:hover{
border: 1px solid #E8907A;
border-radius: 10px;
background:#E8907A;
color: #FFFBDB;
}</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minimize_bu">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>10</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid #707070;
border-radius: 10px;
background:#FFFBDB;
color: #707070;
}
QPushButton:hover{
border: 1px solid #535249;
border-radius: 10px;
background:#535249;
color: #FFFBDB;
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>LOADING</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>close_bu</zorder>
  <zorder>minimize_bu</zorder>
  <zorder>progressBar</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
